data/blog/electron-13-0.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Electron 13.0.0
+author:
+- sofianguy
+- georgexu99
+- VerteDinde
+date: '2021-05-25'
+---
+
+Electron 13.0.0 has been released! It includes upgrades to Chromium `91`, V8 `9.1` and Node.js `14.16.0`. We've added several API updates, bug fixes, and general improvements. Read below for more details!
+
+---
+
+The Electron team is excited to announce the release of Electron 13.0.0! You can install it with npm via `npm install electron@latest` or download it from our [releases website](https://electronjs.org/releases/stable). Continue reading for details about this release, and please share any feedback you have!
+
+## Notable Changes
```

Comment:
I know we haven't fully figured it out yet (at least the electron-rebuild part) but do we want to mention the native module thing here? Depends on whether someone can figure out the electron-rebuild thing today I guess.

Reply:
Working on getting the electron-rebuild part working, I'll add the native module fix here, that's a good callout 🙂

Reply:
Still getting this up here: https://github.com/electron/electron-rebuild/pull/703 I'll leave out of the blog post for now and add in when we add the 91 blog post.